geotrek/signage/templates/signage/sql/post_90_defaults.sql

```diff
@@ -26,6 +26,9 @@ ALTER TABLE infrastructure_infrastructure ALTER COLUMN description SET DEFAULT '
 ALTER TABLE infrastructure_infrastructure ALTER COLUMN published SET DEFAULT FALSE;
 -- publication_date
 -- structure
+ALTER TABLE infrastructure_infrastructure ALTER COLUMN provider SET DEFAULT '';
```

Comment:
It throws errors when I try with `signage_signage`.
I believe this is because `Signage` inherits there 3 fields from `Infrastructure` (https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/signage/models.py#L67) therefore they are actually stored in the infrastructure table..?

Reply:
Odd. I have just checked after applying your migrations : those 3 columns are in the `signage_signage` table so there should be no error. Could you post the error?
Whether the fields are included or not in the child model's table depends on the parent class being abstract (in the Meta): parent abstract class -> columns go in the "child" table (just learned that, thx!).

Reply:
Ok it seems to work actually ! Thanks